### PR TITLE
arangodb 3.5.0 and arangodb 3.4.7

### PR DESCRIPTION
**Pull request by KVS85**
Updated ArangoDB to 3.5.0 and renew for 3.4.7.

### CODE REVIEW

File: library/arangodb [hunk 1]
```diff
@@ -1,12 +1,15 @@
 # maintainer: Frank Celler <info@arangodb.com> (@fceller)
 # maintainer: Wilfried Goesgens <w.goesgens@arangodb.org> (@dothebart)
+# maintainer: Vadim Kondratyev <vadim@arangodb.org> (@KVS85)
```
Comment (yosifkit) [inline library/arangodb]: Just an FYI to @fceller and @dothebart.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index f9d8a08..3a79222 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,5 @@
 arangodb:2.8.11 @ amd64
 arangodb:3.2.17 @ amd64
 arangodb:3.3.23 @ amd64
+arangodb:3.4.7 @ amd64
 arangodb:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 39e899b..e89c44b 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,4 +6,6 @@ arangodb:3.3
 arangodb:3.3.23
 arangodb:3.4
 arangodb:3.4.7
+arangodb:3.5
+arangodb:3.5.0
 arangodb:latest
diff --git a/arangodb_latest/Dockerfile b/arangodb_3.4.7/Dockerfile
similarity index 71%
copy from arangodb_latest/Dockerfile
copy to arangodb_3.4.7/Dockerfile
index 3db2843..e6053a5 100644
--- a/arangodb_latest/Dockerfile
+++ b/arangodb_3.4.7/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:3.8
+FROM alpine:3.10
 MAINTAINER Frank Celler <info@arangodb.com>
 
 ENV ARANGO_VERSION 3.4.7
@@ -7,7 +7,7 @@ ENV ARANGO_PACKAGE arangodb3_${ARANGO_VERSION}-1_amd64.deb
 ENV ARANGO_PACKAGE_URL ${ARANGO_URL}/${ARANGO_PACKAGE}
 ENV ARANGO_SIGNATURE_URL ${ARANGO_PACKAGE_URL}.asc
 
-RUN apk add --no-cache gnupg pwgen nodejs npm binutils && \
+RUN apk add --no-cache gnupg pwgen nodejs npm binutils numactl && \
     npm install -g foxx-cli && \
     rm -rf /root/.npm
 
@@ -33,13 +33,8 @@ RUN cd /tmp                                && \
     echo chgrp 0 /var/lib/arangodb3 /var/lib/arangodb3-apps && \
     echo chmod 775 /var/lib/arangodb3 /var/lib/arangodb3-apps && \
     rm -f /usr/bin/foxx && \
-    wget http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/numactl-2.0.12-r2.apk && \
-    echo "5d6169428e3b8a5d0feda9948a199e9eb676b9a10961f643141f0e462eff38f1  numactl-2.0.12-r2.apk" | sha256sum -c && \
-    apk add ./numactl-2.0.12-r2.apk && \
-    wget http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/numactl-tools-2.0.12-r2.apk && \
-    echo "c758d0ea59a50e2d130ae5df1c35c77da935521ac2649183abde16a6bb1fa4d5  numactl-tools-2.0.12-r2.apk" | sha256sum -c && \
-    apk add ./numactl-tools-2.0.12-r2.apk && \
-    rm -f ${ARANGO_PACKAGE}* data.tar.gz numactl-2.0.12-r2.apk numactl-tools-2.0.12-r2.apk
+    rm -f ${ARANGO_PACKAGE}* data.tar.gz && \
+    apk del gnupg
 # Note that Openshift runs containers by default with a random UID and GID 0.
 # We need that the database and apps directory are writable for this config.
 
diff --git a/arangodb_latest/docker-entrypoint.sh b/arangodb_3.4.7/docker-entrypoint.sh
similarity index 100%
copy from arangodb_latest/docker-entrypoint.sh
copy to arangodb_3.4.7/docker-entrypoint.sh
diff --git a/arangodb_latest/docker-foxx.sh b/arangodb_3.4.7/docker-foxx.sh
similarity index 100%
copy from arangodb_latest/docker-foxx.sh
copy to arangodb_3.4.7/docker-foxx.sh
diff --git a/arangodb_latest/Dockerfile b/arangodb_latest/Dockerfile
index 3db2843..793714c 100644
--- a/arangodb_latest/Dockerfile
+++ b/arangodb_latest/Dockerfile
@@ -1,13 +1,13 @@
-FROM alpine:3.8
+FROM alpine:3.10
 MAINTAINER Frank Celler <info@arangodb.com>
 
-ENV ARANGO_VERSION 3.4.7
-ENV ARANGO_URL https://download.arangodb.com/arangodb34/DEBIAN/amd64
+ENV ARANGO_VERSION 3.5.0
+ENV ARANGO_URL https://download.arangodb.com/arangodb35/DEBIAN/amd64
 ENV ARANGO_PACKAGE arangodb3_${ARANGO_VERSION}-1_amd64.deb
 ENV ARANGO_PACKAGE_URL ${ARANGO_URL}/${ARANGO_PACKAGE}
 ENV ARANGO_SIGNATURE_URL ${ARANGO_PACKAGE_URL}.asc
 
-RUN apk add --no-cache gnupg pwgen nodejs npm binutils && \
+RUN apk add --no-cache gnupg pwgen nodejs npm binutils numactl && \
     npm install -g foxx-cli && \
     rm -rf /root/.npm
 
@@ -33,13 +33,8 @@ RUN cd /tmp                                && \
     echo chgrp 0 /var/lib/arangodb3 /var/lib/arangodb3-apps && \
     echo chmod 775 /var/lib/arangodb3 /var/lib/arangodb3-apps && \
     rm -f /usr/bin/foxx && \
-    wget http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/numactl-2.0.12-r2.apk && \
-    echo "5d6169428e3b8a5d0feda9948a199e9eb676b9a10961f643141f0e462eff38f1  numactl-2.0.12-r2.apk" | sha256sum -c && \
-    apk add ./numactl-2.0.12-r2.apk && \
-    wget http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/numactl-tools-2.0.12-r2.apk && \
-    echo "c758d0ea59a50e2d130ae5df1c35c77da935521ac2649183abde16a6bb1fa4d5  numactl-tools-2.0.12-r2.apk" | sha256sum -c && \
-    apk add ./numactl-tools-2.0.12-r2.apk && \
-    rm -f ${ARANGO_PACKAGE}* data.tar.gz numactl-2.0.12-r2.apk numactl-tools-2.0.12-r2.apk
+    rm -f ${ARANGO_PACKAGE}* data.tar.gz && \
+    apk del gnupg
 # Note that Openshift runs containers by default with a random UID and GID 0.
 # We need that the database and apps directory are writable for this config.
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #6509; 4884cf13; `amd64` (`arangodb`):

```console

$ bashbrew build arangodb:2.8
Using bashbrew/cache:686eba4da6b78bf4604255594b4178dfa1d4983610053da9550154975ea24c50 (arangodb:2.8)
Tagging arangodb:2.8
Tagging arangodb:2.8.11

$ test/run.sh arangodb:2.8
testing arangodb:2.8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build arangodb:3.2
Using bashbrew/cache:644172466c644c03c6c943de861a66ef0c31737882e3a2ee8b872848d33a718e (arangodb:3.2)
Tagging arangodb:3.2
Tagging arangodb:3.2.17

$ test/run.sh arangodb:3.2
testing arangodb:3.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build arangodb:3.3
Building bashbrew/cache:d9e0afeca602efe3c71d178dbd46f8b050fa9e3a4df97049b3fe4626edfa8840 (arangodb:3.3)
Tagging arangodb:3.3
Tagging arangodb:3.3.23

$ test/run.sh arangodb:3.3
testing arangodb:3.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build arangodb:3.4
Building bashbrew/cache:1aa23544f0136378517d88a730744b8155f4136a7ee169bd6f433a735b895b32 (arangodb:3.4)
Tagging arangodb:3.4
Tagging arangodb:3.4.7

$ test/run.sh arangodb:3.4
testing arangodb:3.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build arangodb:3.5
Building bashbrew/cache:428a91c6a856c3e7fbce30e5ac4eedc1e1cca4c4476620ea0c09f785d05d448d (arangodb:3.5)
Tagging arangodb:3.5
Tagging arangodb:3.5.0
Tagging arangodb:latest

$ test/run.sh arangodb:3.5
testing arangodb:3.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

File: library/arangodb (hunk 1 above)
Comment (KVS85) [reply library/arangodb]: Thanks! The guy is described here: https://www.arangodb.com/about-arangodb/.

Comment (dothebart) [reply library/arangodb]: hey @yosifkit - yes, Vadim is overseeing the ArangoDB release management. Sorry we didn't introduce him properly ;-)

**Pull request merged**